lib/activerecord-import/import.rb

```diff
@@ -703,9 +703,13 @@ def import_helper( *args )
             # keep track of the instance and the position it is currently at. if this fails
             # validation we'll use the index to remove it from the array_of_attributes
             arr.each_with_index do |hsh, i|
-              model = new(hsh)
+              model = new do |m|
```

Comment:
@diclophis if this resolves your issue, can you put a comment here why we needed to pass a block to new?

Reply:
will do, i am just now getting through my test suite for this change